Formula/libchewing.rb

```diff
@@ -0,0 +1,48 @@
+class Libchewing < Formula
+  desc "Intelligent phonetic input method library"
+  homepage "http://chewing.im/"
+  url "https://github.com/chewing/libchewing/releases/download/v0.5.1/libchewing-0.5.1.tar.bz2"
+  sha256 "9708c63415fa6034435c0f38100e7d30d0e1bac927f67bec6dfeb3fef016172b"
+  head "https://github.com/chewing/libchewing.git"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "texinfo" => :build
+  depends_on "sqlite" => :build
```

Comment:
>@Chocobo1 To use the system one you remove `depends_on "sqlite" altogether. Does that work locally?

You mean the one installed in system but not from `brew` right?
But what if some user didn't install `sqlite` and just `brew install libchewing`, guess that way the dependency is broken, what is right way to do for this situation?

ps. pardon me, just my first time toying with osx.